bento/src/main/java/com/yelp/android/bento/components/ListComponent.java

```diff
@@ -79,8 +79,8 @@ public void setData(@NonNull List<T> data) {
      * @param data The new data list items to add.
      */
     public void appendData(@NonNull List<T> data) {
-        int oldSize = mData.size();
-        int sizeChange = data.size();
+        int oldSize = mShouldShowDivider ? getTotalSizeWithSeparators(mData.size()) : mData.size();
+        int sizeChange = mShouldShowDivider ? data.size() * 2 : data.size();
```

Comment:
Why not also use `getTotalSizeWithSeparators` here? The last item shouldn't have a divider, so we still need to subtract 1, no?

Reply:
We do not want to subtract 1 in this case. The difference here, is that we need to have a divider _before_ the new items we are inserting, otherwise there would be 2 items touching. 
For example, if we have a list with 2 items and we add 2 items we would have:
```
--item 1--   // index 0
--divider--  // index 1
--item 2--  // index 2

add(2)
--item 1--   // index 0
--divider--  // index 1
--item 2--  // index 2
--divider--  // index 3   <--- this divider is the extra one
--item 3--   // index 4
--divider--  // index 5
--item 4--  // index 6
```
Notice that we would need to call notify (2, 4) **not** (2, 3)